nav2_dwb_controller/dwb_plugins/src/kinematic_parameters.cpp

```diff
@@ -85,46 +85,32 @@ void KinematicParameters::initialize(const std::shared_ptr<rclcpp::Node> & nh)
   setDecelerationAsNeeded(nh, "y");
   setDecelerationAsNeeded(nh, "theta");
 
-  // TODO(crdelsey): Handle dynamic params
-  // the rest of the initial values are loaded through the dynamic reconfigure mechanisms
-  // dsrv_ = std::make_shared<dynamic_reconfigure::Server<KinematicParamsConfig> >(nh);
-  // dynamic_reconfigure::Server<KinematicParamsConfig>::CallbackType cb =
-  //   boost::bind(&KinematicParameters::reconfigureCB, this, _1, _2);
-  reconfigureCB();
+  dsrv_ = std::make_unique<nav2_dynamic_params::DynamicParamsClient>(nh);
+  dsrv_->set_callback([this](auto event){reconfigureCB(event);});
```

Comment:
With the updates in PR #256, you're going to want to call `add_parameters()` before you set the callback or the callback will not be triggered since there is a filter for dynamic parameters.